types/aws-lambda/index.d.ts

```diff
@@ -39,6 +39,7 @@
 //                 Peter Savnik <https://github.com/savnik>
 //                 Sven Milewski <https://github.com/svenmilewski>
 //                 Benoit Boure <https://github.com/bboure>
+//                 James Lakin <https://github.com/jamesorlakin>
```

Comment:
Be aware this will ping you on all PRs for this package - but glad to have you if you want to help!

(I've been thinking about moving the authors that haven't responded to PRs at all in the last few years to a separate contributors section so we don't have as much of a bystander effect on PRs)

Reply:
Yeah I don't mind getting stuck in! I develop with Lambda quite frequently so the odd PR review would be fine 🙂